Jetty 12.0.x document modules (#9151)
eclipse/jetty.project: documentation/jetty-documentation/src/main/asciidoc/operations-guide/modules/module-resources.adoc

//
// ========================================================================
// Copyright (c) 1995-2022 Mort Bay Consulting Pty Ltd and others.
//
// This program and the accompanying materials are made available under the
// terms of the Eclipse Public License v. 2.0 which is available at
// https://www.eclipse.org/legal/epl-2.0, or the Apache License, Version 2.0
// which is available at https://www.apache.org/licenses/LICENSE-2.0.
//
// SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
// ========================================================================
//

[[og-module-resources]]
===== Module `resources`

include::{JETTY_HOME}/modules/resources.mod[tags=description]

A common use-case for this module is to provide resources for third-party libraries via the xref:og-start-start-class-path[server classpath].

> For instance, many logging libraries (including https://logging.apache.org/log4j/2.x/[log4j] and https://logback.qos.ch/[Logback]) look for their configuration files on the classpath.

Reviewer: log4j -> Log4j2

> In fact, Jetty's own logging library -- enabled via the `logging-jetty` module -- stores its configuration in `$JETTY_BASE/resources/jetty-logging.properties`.

Reviewer: ```suggestion
Jetty provides a logging library implementation -- enabled via the `logging-jetty` module -- whose configuration file is `$JETTY_BASE/resources/jetty-logging.properties`.
```
